share/workspaces/Advanced.xml

```diff
@@ -2211,6 +2211,13 @@
             <label>coda</label>
             </Marker>
           </Cell>
+        <Cell name="To Coda">
+          <Marker>
+            <style>Repeat Text Left</style>
+            <text>To <font size="20"/><sym>coda</sym></text>
```

Comment:
Unfortunately we have to hard code the size of that symbol (here to the default size of the 'plain' Coda, 20 pt), it'd be way too small otherwise (12 pt per default).